plugins/analytics-module-ga/config.d.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  app: {
+    // TODO: Only marked as optional because backstage-cli config:check in the
+    // context of the monorepo is too strict. Ideally, this would be marked as
```

Comment:
Yeah this ends up just being an unfortunate thing with us wanting to have multiple analytics providers installed in the repo at once 😅 Perhaps that's not completely insane for other projects too though? Could be good to make sure the config schema is structured in a way that allows for multiple providers? With that I mean we should remove the `provider: 'ga'` as it'd conflict and is probably redundant anyway? I'm thinking if `analytics.ga` exists, then it is enabled.

Having looked through this I realize that `app.analytics.provider` is actually unused right? In which case it should be nuked from orbit